easybuild/framework/easyblock.py

```diff
@@ -315,7 +317,10 @@ def fetch_patches(self, list_of_patches, extension=False, checksums=None):
                         copy_file = True
                     suff = patch_entry[1]
                 else:
-                    self.log.error("Wrong patch specification '%s', only int and string are supported as second element!" % patch_entry)
+                    self.log.error(
+                        "Wrong patch specification '%s', only int and string are supported as second element!",
```

Comment:
it's stupid, we already have enough line of code to scroll around in, there's absolutely no reason why this should be spread across 4 lines of code, it's madness